esp-hal/src/twai/mod.rs

```diff
@@ -783,7 +787,22 @@ where
         baud_rate: BaudRate,
         interrupt: Option<InterruptHandler>,
     ) -> Self {
-        Self::new_internal(peripheral, tx_pin, rx_pin, clocks, baud_rate, interrupt)
+        Self::new_internal(
+            peripheral, tx_pin, rx_pin, clocks, baud_rate, interrupt, false,
+        )
+    }
+
+    pub fn new_no_transceiver<TX: OutputPin, RX: InputPin>(
```

Comment:
I'm just curious and I don't know anything about TWAI: Why this expects `tx` pin?
Also, could you please add a doc string here?